src/Language/PureScript/CodeGen/JS.hs

```diff
@@ -69,9 +69,9 @@ moduleToJs (Module _ coms mn _ imps exps reExps foreigns decls) foreignInclude =
           . filter (flip S.member usedModuleNames)
           . (\\ (mn : C.primModules)) $ ordNub $ map snd imps
     F.traverse_ (F.traverse_ checkIntegers) optimized
-    comments <- not <$> asks optionsNoComments
+    comments <- asks (not . optionsNoComments)
```

Comment:
I'm pretty sure this change has nothing to do with this PR, right? Just a stylistic change you happened to make?

Reply:
Correct. This was an hlint suggestion to get rid of the unnecessary `fmap`, I believe.

Reply:
I forgot to mention I wasn't sure if I could/should add this to my PR. I was going to ask and ended up forgetting.

Reply:
I think leaving it in is fine.